docs/running/tardis_widgets/generating_widgets.ipynb

```diff
@@ -4,18 +4,24 @@
    "cell_type": "markdown",
```

Comment:
<p>By selecting a wavelength range on the spectrum plot, you can see the species responsible for producing the features within that range. This is determined by counting the number of escaping packets that experienced their last interaction with each ion. Packets can be filtered by the wavelength at which they were absorbed or emitted. You can then click on each species to see the packets counts for specific lines. These can be grouped in several ways. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1280/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>